invenio_workflows/manage.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2015 CERN.
+#
+# Invenio is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+"""Perform operations with workflows."""
+
+from __future__ import print_function
+
+import argparse
+import sys
+
+from flask import current_app
+
+from invenio.ext.script import Manager
+
+import six
+
+from werkzeug.utils import import_string
+
+from .api import (
+    start as start_workflow,
+    start_delayed
+)
+
+manager = Manager(description=__doc__)
+
+
+def split_marcxml(source):
+    """Split a MARCXML file using dojson MARC21 utils."""
+    from dojson.contrib.marc21.utils import split_blob
+    return [data for data in split_blob(source.read())]
+
+
+@manager.option('source', type=argparse.FileType('r'), default=sys.stdin,
```

Comment:
`inveniomanage workflows start NAME [-i [STDIN]] [-t [json]] [--delayed]`

I would just swapped `workflow_name` and `source` arguments. WDYT?